### PR TITLE
Fixing index.md syntax

### LINKED ISSUE #1780: Some front page formatting is off (HTML tags in there)
The formatting on come pages are off in production. In general, there are HTML tags that should not be there.
- [x] Rackspace CDN
- [x] Cloud Big Data
- [x] Cloud Servers
- [x] Managed VMWare Services
- [x] Cloud Images
- [x] Cloud Sites
- [x] Cloud Load Balancers
- [x] Rackspace RackConnect
- [x] Cloud Files
- [x] Cloud Block Storage
- [x] Cloud Databases
- [x] Cloud Queues
- [x] Cloud Backup
- [x] Rackspace Monitoring
- [x] Cloud Orchestration
- [x] Rackspace Auto Scale
- [x] Rackspace Intelligence
- [x] Rackspace Metrics
- [x] RPCO
- [x] RPC Microsoft

### PR DESCRIPTION
**Pull request by nimbinatus**
fixing list formatting

Fixes #1780

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4111200ac9/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-68943fba32/how-to/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-987eb587b2/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e0bbf810e8/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c13d0d441e/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-13f8563215/how-to/)

File: content/sharepoint/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ product: Microsoft SharePoint
 product_url: sharepoint
 ---
 
-<p class="lead" markdown="1">Our hosted SharePoint service offers all of the benefits of an online SharePoint site plus access to Rackspace's renowned Fanatical Support.</p>
+Our hosted SharePoint service offers all of the benefits of an online SharePoint site plus access to Rackspace's renowned ***Fanatical Support***&reg;.
```
Comment (stephamon) [inline content/sharepoint/index.md]: This is the most nitpicky thing ever, but you only need two asterisks to bold text. Three isn't going to break anything, but I figured we should probably be consistent.

Comment (nimbinatus) [inline content/sharepoint/index.md]: But we're supposed to use bold italic for Fanatical Support, aren't we? I thought that's what the style guide says... (ref http://rackerlabs.github.io/docs-rackspace/style-guide/m-z-style-guidelines.html#trademarks)

Comment (stephamon) [inline content/sharepoint/index.md]: Markdown doesn't allow for nested text style, unfortunately. You can either do bold or italic, but not both. RST allows you to have both, but Markdown won't. At least GitHub flavored Markdown doesn't.

Comment (nimbinatus) [inline content/sharepoint/index.md]: @stephamon, odd. I've had it work with that construction. Maybe it's this specific syntax? Here's an example: https://support.rackspace.com/how-to/managed-cloud-service-levels-faq/ (look under "What do I get with Fanatical Support?")

Comment (stephamon) [inline content/sharepoint/index.md]: Oh how interesting! I've never seen that work before. I wonder if GitHub made an update to their flavored markdown. Let's see how it turns out!

**Comment by stephamon** [comment]
👍

**Pull request merged**